manpages.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"fmt"
+	"regexp"
+	"strconv"
+	"strings"
+)
+
+var manpagesMatcher = regexp.MustCompile(`\b(\w+)\((\d+)\)(\W|$)`)
```

Comment:
Yes: section names use dashes and underscores. `[0-9][0-9a-z_-]*` matches all current section names, see https://play.golang.org/p/2JytW6o-p_